buildconfig/stubs/gen_consts.py

```diff
@@ -0,0 +1,23 @@
+"""
```

Comment:
I don't think this should be run along with format because it's not a formatting change. I think it should be something run manually by a user. It's not possible to forget doing this because of stubtest